### ISSUE TITLE
Recommended Batch Size and Training Steps for SFT of CogVideoX-2B

### ISSUE THREAD

**Issue by aHapBean** [THUDM/CogVideo#733]
Hi. I am fine-tuning CogVideoX-2B with 10,000 video samples and would like to know the recommended batch size and training steps for supervised fine-tuning (SFT).

**Comment by OleehyO** [comment]
It's hard to say exactly how much batch size and computation are needed, but typically if you find that the training loss is hard to decrease or the grad norm is very steep, then you may need to increase the batch size

**Comment by aHapBean** [comment]
Thanks a lot for your response!
